specs/architecture.md

```diff
@@ -55,7 +53,25 @@ User interface needs also some data visualizations of all the data so the [data2
 
 For the RESTful API endpoints generation in Android app the [Kotlin code generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/kotlin.md) is used.
 
-The client app is designed to be multilanguage app  to provide access to occupancy data also for foreign people. All the translations are managed via resource files with maapings for application text views.
+If needed custom widgets can be designed by developers team or used some open-source alternatives to standard Android widgets from web.
```

Comment:
This sentence is hard to parse. Could you rephrase?

Reply:
tried to fix